tests/generic_views/views.py

```diff
@@ -66,6 +66,12 @@ class AuthorList(generic.ListView):
     queryset = Author.objects.all()
 
 
+class AuthorListGetQueryset(AuthorList):
+    def get_queryset(self):
+        """A get_queryset that doesn't return anything"""
+        pass
```

Comment:
Perhaps be explicit and return `None` here.

Reply:
Not sure. I can see it could make sense, but the case I'm trying to capture is when `get_queryset` doesn't return anything, rather than explicitly `None`. I ran into this with a `get_queryset` where it returned different `QuerySet`s based on the logged in user's group - but didn't have a fallback so it gave a cryptic error for anonymous users and users not belonging to a group. Maybe I can show that better with a condition that doesn't pass?

```qs = super(AuthorListGetQueryset, self.get_queryset()
if user.is_superuser:
    return qs
```

... with some better commenting to explain the thought behind it?

Or I just return `None` and be done with it.

Reply:
Python functions implicitly return `None` if return is not called. I feel being explicit is better for a test.

Reply:
Ok, agreed